webapp/src/components/cardDetail/comment.test.tsx

```diff
@@ -14,10 +14,16 @@ import {TestBlockFactory} from '../../test/testBlockFactory'
 
 import mutator from '../../mutator'
 
+import {mockMMStore} from '../../../tests/mock_window'
+
 import Comment from './comment'
 
 jest.mock('../../mutator')
 const mockedMutator = mocked(mutator, true)
+jest.mock('../../webapp_globals', () => ({
+    ...jest.requireActual('../../webapp_globals'),
+    messageHtmlToComponent: jest.fn(() => <div className="mocked-message-html">Test Comment</div>),
```

Comment:
This isn't part of this PR, but when I saw this, I realized that whenever we get to shipping a plugin API package from the web app, it should include mock versions of all the APIs to make writing tests that use them easier.

In the mean time, you could add this to the Jest setup script to avoid having to redefine it in multiple test files

Reply:
@hmhealey Looking at the repo now, I realize that `jest.config.js` isn’t configured in the board/focalboard repo. I’m planning to set up the proper Jest configuration in a follow-up PR, which will include the changes mentioned above.

Reply:
It's actually configured as part of the `package.json` currently, but moving it out to it's own file and setting it up a bit better makes sense to me. While it's nice to use fewer files sometimes, I prefer having it spread out since the `package.json` already does so much stuff

Reply:
Ah, got it. Done.